sdk/tables/azure-data-tables/src/private/package_version.hpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @file
+ * @brief Provides version information.
+ */
+
+#pragma once
+
+#define AZURE_DATA_TABLES_VERSION_MAJOR 12
+#define AZURE_DATA_TABLES_VERSION_MINOR 2
+#define AZURE_DATA_TABLES_VERSION_PATCH 0
+#define AZURE_DATA_TABLES_VERSION_PRERELEASE "beta.1"
```

Comment:
We should think about the package versioning story for Tables. I assume it will be independent of `Azure::Storage`, and hence wouldn't start from **12.2.0**.

We should look at other languages and align versioning with where they started (based on their rationale).